projects/1.19/assets/ars-armiger/ars_armiger/lang/zh_cn.json

```
@@ -124,7 +124,7 @@
     "ars_armiger.page.tetra_weapons.page3.body": "在模块化武器上嵌入法术核心后，武器便会在击中目标或掷中方块时对其施放抄入的法术。可在抄写台上抄入法术，抄入武器的$(l)不可/$包含形态类构件，例如“点燃 -> 延长时效”。",
     "ars_armiger.page.tetra_weapons.page4.body": "法术打击有 3 个核心属性：强度、使用率、冷却。$(br)§n强度§r是从右往左额外加入的强化之魔符的数目。由此拥有 3 个效果类构件的法术需要强度 3 以强化所有效果。$(br)§n冷却§r是每次法术打击使用的时间间隔，单位为秒。$(br)§n使用率§r是施放法术时魔力消耗量的倍数。以上三种属性均可通过打磨强化！",
     "ars_armiger.page.tetra_weapons.page5.title": "元素核心",
-    "ars_armiger.page.tetra_weapons.page5.body": "除奥术核心外，你也可以用元素精华嵌入元素核心。这会将武器与某一元素调谐。调谐后的武器会为所有同学派的效果类构件附加强化，并为其他元素学派的效果类构件附加 1 个抑制之魔符。",
+    "ars_armiger.page.tetra_weapons.page5.body": "除奥术核心外，你也可以用嵌入元素精华，形成元素核心。这会将武器与某一元素调谐。调谐后的武器会为所有同学派的效果类构件附加“强化”，并为其他元素学派的效果类构件附加 1 个抑制之魔符。",

```

```suggestion
    "ars_armiger.page.tetra_weapons.page5.body": "除奥术核心外，你也可以嵌入元素精华，形成元素核心。这会将武器与某一元素调谐。调谐后的武器会为所有同学派的效果类构件附加“强化”，并为其他元素学派的效果类构件附加 1 个抑制之魔符。",
```
[考量] 疑为笔误，但不知道这么改对不对

---

是删漏了（